modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-annotations.adoc
+
+[id="serverless-functions-adding-annotations_{context}"]
+= Adding annotations to a function
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every annotation that you want to add, add the following YAML to the `annotations` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+annotations:
+  color: "red"
+----
++
+Substitute `color: "red"` with your annotation.
+
+. Save the configuration.
+
+The next time you propagate the configuration to the cluster, the corresponding service will have the added annotations.
```

Comment:
```suggestion
The next time you deploy your function to the cluster, the annotations are added to the corresponding service.
```

Reply:
@lance How about keeping "propagate the configuration", because we might be re-deploying the function?

Reply:
> @lance How about keeping "propagate the configuration", because we might be re-deploying the function?

The sentence as it is actually implies a redeployment with the words "the next time you...".

But my real concern is that we haven't really talked about "propagating the configuration" anywhere in any of these documents. We only talk about deploying. So as a new developer, this might confuse me. It may be a common phrase among seasoned Kubernetes developers. But those developers aren't the primary audience. I would not be opposed to this wording if we state somewhere what "propagate the configuration" means (maybe in the `deploy` docs).

Reply:
I'd go with Lance's wording, deploy IMO is more descriptive / easier to understand than "propagate"

Reply:
Good points, implemented.